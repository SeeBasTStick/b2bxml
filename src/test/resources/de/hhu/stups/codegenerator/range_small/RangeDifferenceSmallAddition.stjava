public static void main(String[] args) {
    RangeDifferenceSmall range = new RangeDifferenceSmall();
    range.simulate();
    System.out.println(range.set.card());
}