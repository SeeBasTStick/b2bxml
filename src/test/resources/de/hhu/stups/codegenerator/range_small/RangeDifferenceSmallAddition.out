public static void main(String[] args) {
    RangeComplementSmall range = new RangeComplementSmall();
    range.simulate();
    System.out.println(range.set);
}